### ISSUE TITLE
Can't install with `apt`: `trying to overwrite '/usr/.crates2.json', which is also in package bat`

### ISSUE THREAD

**Issue by MPvHarmelen** [BurntSushi/ripgrep#1777]
#### What version of ripgrep are you using?

Not applicable.

#### How did you install ripgrep?
I tried installing using `apt`:

```
$ sudo apt install ripgrep
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed
  ripgrep
0 to upgrade, 1 to newly install, 0 to remove and 34 not to upgrade.
Need to get 0 B/1.228 kB of archives.
After this operation, 4.502 kB of additional disk space will be used.
(Reading database ... 202335 files and directories currently installed.)
Preparing to unpack .../ripgrep_11.0.2-1build1_amd64.deb ...
Unpacking ripgrep (11.0.2-1build1) ...
dpkg: error processing archive /var/cache/apt/archives/ripgrep_11.0.2-1build1_amd64.deb (--unpack):
 trying to overwrite '/usr/.crates2.json', which is also in package bat 0.12.1-1build1
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Errors were encountered while processing:
 /var/cache/apt/archives/ripgrep_11.0.2-1build1_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

#### What operating system are you using ripgrep on?

64-bit Ubuntu 20.04.1LTS

#### What is the expected behavior?

To install smoothly.

**Comment by BurntSushi** [comment]
This is an issue with the Debian packages you're using: https://bugs.launchpad.net/ubuntu/+source/rust-bat/+bug/1868517

This is not an issue with ripgrep itself. The ripgrep project has nothing to do with the Debian packages for ripgrep.

**Comment by modus-jose** [comment]
This has been fixed on Ubuntu 20.10
For a solution on Ubuntu 20.04 for example to install ripgrep
sudo apt-get -qq install -o Dpkg::Options::="--force-overwrite" bat ripgrep
